db/queries/user.sql

```diff
@@ -1,3 +1,57 @@
+-- name: UserGetInfo :one
+
+SELECT
+    "id",
+    "name",
+    "email",
+    "image_id",
+    "enabled"
+FROM "user" u
+WHERE u."username" = $1;
+
+-- name: UserUpdateInfo :one
+
+UPDATE "user"
+SET
+    "name" = COALESCE($2, "name"),
+    "email" = COALESCE($3, "email"),
+    "address" = COALESCE($4, "address"),
+    "image_id" = COALESCE($5, "image_id")
+WHERE "username" = $1
+RETURNING
+    "id",
+    "name",
+    "email",
+    "image_id",
+    "enabled";
+
+-- name: UserUpdatePassword :one
+
+UPDATE "user"
+SET
+    "password" = sqlc.arg(new_password)
+WHERE
+    "username" = $1
+    AND "password" = sqlc.arg(current_password)
```

Comment:
This would fail because passwords are hashed and salted using `bcrypt`, password checking should be done on server side